connect-examples/server-library-example/client/src/main/scala/org/example/Main.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.example
+
+import java.nio.file.{Path, Paths}
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.types.{StructType, StructField, StringType, IntegerType}
+import org.example.proto
+import org.example.proto.CreateTable.Column.{DataType => ProtoDataType}
+import com.google.protobuf.Any
+import org.apache.spark.connect.proto.Command
+
+object Main {
```

Comment:
Shall we have some meaningful name instead of `org.example.Main`?

Reply:
Absolutely, changed this to `CustomTableExample`